package.json

```diff
@@ -1,29 +1,31 @@
 {
-  "name": "dmt",
-  "version": "0.1.0",
-  "description": "Project management tool",
-  "license": "GPL-2.0+",
-  "scripts": {
-    "test": "karma start"
-  },
-  "devDependencies": {
-    "casperjs": "~1.1.0-beta3",
-    "karma": "~0.13.15",
-    "karma-junit-reporter": "~0.3.8",
-    "karma-html2js-preprocessor": "~0.1.0",
-    "karma-phantomjs-launcher": "~0.2.1",
-    "qunitjs": "~1.17.1",
-    "karma-qunit": "~0.1.8",
-    "requirejs": "~2.1.21",
-    "karma-requirejs": "~0.2.2",
-    "karma-sinon": "~1.0.4",
-    "phantomjs": "~1.9.18",
-    "sinon": "~1.17.2",
-    "sinon-qunit": "~2.0.0"
-  },
-  "repository": {
-    "type": "git",
-    "url": "git://github.com/ccnmtl/dmt.git"
-  },
-  "readmeFilename": "README.markdown"
+    "name": "dmt",
```

Comment:
Just to note: I was using 2 spaces for indentation here because that's how npm formats package.json when doing `npm install some-package --save` and is not likely to change (see: https://github.com/npm/npm/pull/3062)

But I'm fine with using 4 spaces here like we do for everything else, and just maintaining the file manually, since the benefits of being able to use the `--save` and `--save-dev` options for npm install are not that much.